test/Fondue.test.js

```diff
@@ -128,15 +128,15 @@ describe("fromDataBuffer", () => {
 	});
 });
 
-describe("format", () => {
+describe("outline format", () => {
 	test("OTF font", async () => {
 		const fondue = await SourceCodeProOTFFont();
-		expect(fondue.format).toBe("OpenType/CFF");
+		expect(fondue.outlines).toContain("OpenType CFF");
```

Comment:
I would prefer this test to be more precise, e.g. `toBe(["OpenType CFF"])`. Could you also add a test that verifies behavior of a single font with multiple outlines?

Reply:
I don't even know if these fonts really exist 😅 I could whip up a custom test font for this, but that maybe sounds like overkill?

Also Jest suggested `.toStrictEqual` instead of `.toBe` so I used that!

Reply:
Ah if it's difficult to get sample data for this I wouldn't bother with it!